packages/strapi-admin/admin/src/containers/PluginDispatcher/index.js

```diff
@@ -46,7 +47,10 @@ export function PluginDispatcher(props) {
 
   return (
     <div>
-      <Helmet title={`Strapi - ${name}`} />
+      <Helmet
```

Comment:
We should create a component for the Helmet it will prevent the double imports of the favicon, it will also be easier to customise for the users.